docker/1.4.0/final/Dockerfile.eia

```diff
@@ -0,0 +1,41 @@
+ARG py_version
+
+# TODO: the base image name may change
```

Comment:
some of your TODOs aren't particularly actionable

Reply:
removed